time epoch
uint16 pose_x
uint16 pose_y
uint16 theta


